package.json

```diff
@@ -69,8 +69,11 @@
     "@apollo/gateway": "2.1.1",
     "@apollo/subgraph": "2.1.1",
     "@apollo/utils.keyvadapter": "3.0.0",
+    "@aws-sdk/client-s3": "3.637.0",
```

Comment:
I'm wondering about `@aws-sdk/...` versions if we should update `@aws-sdk/client-sqs` and `@aws-sdk/credential-provider-node` so all `@aws-sdk` versions is matching.

Reply:
I've upgraded the credentials provider, but the client-sqs has multiple breaking changes between 3.1 and 3.6.
I fixed a few simple issues before hitting some more complex ones that would require more investigation and probably refactoring 🤔

Reply:
Ok looked into it some more and think I managed to solve all the issues with the sqs upgrade, feel pretty confident after I noticed that the only place the issue was was just in a log statement 😅